deploy/sre-prometheus/100-sre-node-spacefillingup.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: sre-node-filedescriptor-limit
+    role: alert-rules
+  name: sre-node-filedescriptor-limit
+  namespace: openshift-monitoring
+spec:
+  groups:
+  - name: sre-controlplane-node-almostoutofspace
+    rules:
+    - alert: NodeFilesystemAlmostOutOfSpaceSRE
+      # This is the same as the upstream alert, but groups by instance id (which is which node is affected)
+      # and then only fires on the control-plane, infra, or master node roles
+      # See the ocm-agent folder for the WorkerNode version of this alert
+      annotations:
+        description: Filesystem on {{ $labels.device }} at {{ $labels.instance }}
+          has only {{ printf "%.2f" $value }}% available space left.
+        runbook_url: https://github.com/openshift/runbooks/blob/master/alerts/cluster-monitoring-operator/NodeFilesystemAlmostOutOfSpace.md
+        summary: Filesystem has less than 3% space left.
+      expr: |-
+        group by (instance)
```

Comment:
I think if we remove this `group by (instance)` line it should still work, I tested this with the other NodeFilesystemSpaceFillingUp alert, and that ends up preserving the labels on the alert itself while only showing the ones firing on whatever nodetype is defined under `* on (instance)` and below

Reply:
Good idea to keep the details that we can get from the metrics.